lib/smart_proxy_openscap/storage_fs.rb

```diff
@@ -26,6 +26,13 @@ def get_arf_html(digest)
       arf_object.html.force_encoding('UTF-8')
     end
 
+    def delete_arf_file
+      path = "#{@path_to_dir}/#{@namespace}/#{@cname}/#{@id}"
```

Comment:
Why not use [`@path`](https://github.com/xprazak2/smart_proxy_openscap/blob/delete-arf-report/lib/smart_proxy_openscap/storage.rb#L16) ?